### PR TITLE
Marcele silv patch 1

### PR DESCRIPTION
**Pull request by MarceleSilv**

### CODE REVIEW

**Comment by MarceleSilv** [comment]
Hello, I would like to make this change to complete the inclusion of Dark Mode. 🙏

File: README.MD [hunk 1]
```diff
@@ -48,6 +48,7 @@ Please note these demos should be considered as CSS "Proofs of Concepts". They m
 1. [Ripple Effect](#RippleEffect)
 1. [Responsive Counter Showing # of Items That Didn't Fit Screen](#ResponsiveItemShowcase)
 1. [Neumorphism Card Design](#neumorphism-card-design)
+1. [Dark Mode](#DarkMode)
```
Comment (aayush105) [inline README.MD:51]: Make it alphabetical order

**Review by aayush105 — COMMENTED** [review]
@MarceleSilv there is a conflict resolve it and i have requested a change fix it and push it again.
To resolve the conflict, first update the branch,pull the repo and push your corrected project code again.

**Review by gaurovgiri — APPROVED** [review]
Seems good to me!

**Pull request closed without merging**